source/installation-latex-admin.Rmd

```diff
@@ -0,0 +1,50 @@
+---
+title: "LaTeX installeren als administrator"
+maintainer: "Thierry Onkelinx"
+---
+
+## Windows
+
+De installatiebestanden zijn beschikbaar via http://miktex.org/download
+
+1. Start de installer en klik op _Uitvoeren_.
+1. Aanvaard de voorwaarden en klik op _Volgende_
+1. Selecteer _Anyone who uses this computer_ en klik op _volgende_.
+1. In het veld _Install MikTeX to_ vul je `c:\R\MikTeX` in en klik op _volgende_.
+1. Wijzig _Install missing packages on-the-fly_ naar `Yes` en klik op _volgende_.
+1. Klik op _Start_.
+1. Klik op _volgende_.
+1. Klik op _close_.
+1. Ga in het _Start_ menu naar de _MikTex_ map.
+1. Start _Update (admin)_ in de submap _Maintainance (admin)_.
+1. Selecteer _remote package repository_ en _nearest package repository_ en klik tenslotte op _Volgende_.
+1. Klik op _Volgende_.
+1. Klik op _Volgende_.
+
+### Afwijkingen t.o.v. default installatie
+
+- Installeren naar `C:\R\MikTex` i.p.v. `C:\Program Files\MikTex`
+- _Install missing packages on-the-fly_: `Yes`
+
+## Ubuntu
+
+### Minimale installatie
+
+```
+sudo apt-get update
+sudo apt-get install -y --no-install-recommends \
+    lmodern \
+    qpdf \
+    texlive-fonts-recommended \
+    texlive-humanities \
+    texlive-latex-extra \
+    texinfo
+cd /usr/share/texlive/texmf-dist
+sudo wget http://mirrors.ctan.org/install/fonts/inconsolata.tds.zip
+sudo unzip inconsolata.tds.zip
+sudo rm inconsolata.tds.zip
+sudo echo "Map zi4.map" >> /usr/share/texlive/texmf-dist/web2c/updmap.cfg
+sudo mktexlsr
+sudo updmap-sys
+cd ~
+```
```

Comment:
texlive on Ubuntu tends not to get updated. You may consider pieces of this [README.md](https://github.com/inbo/INBOmd/raw/templates_changed/README.md) for instructions to manually install texlive from CTAN in Ubuntu, after having it installed with `apt-get`.